### ISSUE TITLE
relative_train_NCEV.txt

### ISSUE THREAD

**Issue by LeoWade001** [ZebangCheng/Emotion-LLaMA#5]
<img width="962" alt="截屏2024-07-20 10 48 50" src="https://github.com/user-attachments/assets/58ed5000-d759-4208-b637-056c21f8d083">
hi，author，with the help of your article and questions, I successfully reproduced the results, but I encountered this file missing in the process of fine-tuning. I want to ask if this file is the same as the relative_test3_NCEV.txt in the previous question. If not, can you request to provide this file? I would appreciate it if you helped me.

**Comment by ZebangCheng** [comment]
I'm not sure which dataset you are using for instruction tuning. If you are tuning on your own dataset, you need to create a file in the format of relative_train_NCEV.txt based on your dataset. If you are tuning on the MER2023 dataset, you need to apply to use the dataset from the MER2023 official source and then create the relative_train_NCEV.txt file based on the annotation files within the dataset.

Please note that in the file name relative_test3_NCEV, N stands for the name of the video sample (e.g., samplenew_00006611), C represents the count of video frames (125, this variable is used elsewhere for hierarchical frame extraction. For instance, when extracting VideoMAE features, this project sets an input of 16 images. The 125 video frames are sequentially divided into eight parts, and two frames are randomly selected from each part as input), and E and V represent the emotion labels (sad and -10). In this project, C and V values are not actually used. You can create your training data file with just the name and label. Then, modify the corresponding dataset reading code to use this file.

**Comment by LeoWade001** [comment]
Thank you for your help in asking me questions. Now I am tuning the MER2023 data set, and I am currently in the application stage. I look forward to contacting and communicating with you if I have any questions.

**Comment by LeoWade001** [comment]
Thank you for your help in my questions. Now I am tuning the MER2023 data set, and I am currently in the application stage. I look forward to contacting and communicating with you if I have any questions.
